resources/assets/lib/core/wiki/reference-link-tooltip.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+export default class ReferenceLinkTooltip {
+  constructor() {
+    $(document).on('mouseover', '.js-reference-link', this.showTooltip);
+  }
+
+  private createTooltip = (element: HTMLElement, content: HTMLElement) => {
+    $(element).qtip({
+      content: {
+        text: content,
```

Comment:
this only looks correct by accident as `--paragraph-space` isn't set on any of the parents.